backend/blueprints/spa_api/service_layers/replay/tag.py

```diff
@@ -153,8 +154,7 @@ def decode_tag(encoded_key: str):
 def apply_tags_to_game(query_params: Dict[str, any] = None, game_id=None, session=None):
```

Comment:
is `apply_tags_to_game` used elsewhere without the `query_params` kwarg?

Reply:
No but the value can be none when passed in.
This makes that ability for it to be none explicit.

The other values are optional though.